app/uploaders/ckeditor_attachment_file_uploader.rb

```diff
@@ -0,0 +1,38 @@
+# encoding: utf-8
+require 'carrierwave'
+
+class CkeditorAttachmentFileUploader < CarrierWave::Uploader::Base
+  include Ckeditor::Backend::CarrierWave
+
+  # Include RMagick or ImageScience support:
+  # include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+  # include CarrierWave::ImageScience
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
```

Comment:
keep in mind that :file storage will not work on heroku
You need to setup Amazon S3 and use it when app in production environment